authors/apps/articles/serializers.py

```diff
@@ -150,7 +153,27 @@ def create(self, validated_data):
         )
 
 
-
 class ShareArticleSerializer(serializers.Serializer):
     content = serializers.CharField()
     share_with = serializers.CharField()
+class RateSerializer(serializers.ModelSerializer):
```

Comment:
@mendozabree , please add two lines before this new class.

Reply:
Thanks for pointing that out, I have worked on it!